Library/Homebrew/brew.rb

```diff
@@ -136,17 +136,17 @@
   end
 rescue UsageError => e
   require "help"
-  Homebrew::Help.help cmd, remaining_args: args.remaining, usage_error: e.message
+  Homebrew::Help.help cmd, remaining_args: T.must(args).remaining, usage_error: e.message
```

Comment:
```suggestion
  Homebrew::Help.help cmd, remaining_args: args&.remaining, usage_error: e.message
```
or something seems a bit safer here given the context here.